instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import Comment from '../CommentSection/Comment';
+import PostInteraction from '../PostInteraction/PostInteraction';
+import propTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import '../CommentSection/commentsection.css';
+
+class CommentSection extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: props.comments,
+      newComment: '',
+      likes: this.props.likes
+    };
+  }
+
+  addNewComment = event => {
+    event.preventDefault();
+    this.setState({
+      comments: [
+        ...this.state.comments,
+        { text: this.state.newComment, username: 'Brandon' }
+      ],
+      newComment: ''
+    });
+  };
+
+  onChangeHandler = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  addLikes = () => {
+    this.setState({ likes: this.state.likes + 1 });
```

Comment:
It's a bit odd that likes just keeps going up. Maybe switch to like +1 or -1 toggle.